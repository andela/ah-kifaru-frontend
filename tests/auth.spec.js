import React from 'react';
import toJson from 'enzyme-to-json';
import { mount } from './enzyme';
import Auth from '../src/components/auth';


describe('Application test', () => {
  it('should renders without crashing', () => {
    const wrapper = mount(<Auth />);
    expect(toJson(wrapper)).toMatchSnapshot();
    expect(wrapper.find('a').length).toBeGreaterThan(1);
    expect(wrapper.find('a')).toHaveLength(2);
    expect(wrapper.find('[href="/login"]')).toHaveLength(1);
    expect(wrapper.find('[href="/signup"]')).toHaveLength(1);
  });
});