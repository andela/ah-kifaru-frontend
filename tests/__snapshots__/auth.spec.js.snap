// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application test should renders without crashing 1`] = `
<cards>
  <div
    className="flex justify-center w-full md:hidden "
  >
    <div
      className="max-w-sm flex items-center h-screen overflow-hidden w-full"
    >
      <div
        className="flex justify-center items-center h-screen flex-col w-full"
      >
        <div
          className="flex flex-col items-center"
        >
          <a
            className="bg-teal-500  text-white text-center text-xs w-32  uppercase  py-3 px-5 "
            href="/login"
          >
            LOGIN
          </a>
          <a
            className="bg-white-500 mt-6  text-center border border-red-500  w-32 text-xs py-3 px-1 uppercase text-red-600"
            href="/signup"
          >
            get started
          </a>
        </div>
      </div>
    </div>
  </div>
</cards>
`;
